client/pages/Interests/index.jsx

```diff
@@ -32,6 +40,42 @@ class Interests extends React.Component {
       interests,
     });
   }
+
+  queryCalendarUrl = () => {
+    this.props.getCalendarUrl()
+     .then(authUrl => window.location.href = authUrl);
```

Comment:
What happens here when an error (say network) occurs? `authUrl` may be undefined?

Reply:
Thanks for pointing this out but error case has been taken care of in the action and a toaster has been in case of error @olusoladavid

Reply:
As for the undefined part I will take care of that now